sshuttle/client.py

```diff
@@ -643,9 +651,23 @@ def matches_acl(dstip, dstport, store_to_check):
 
     return False
 
+def ipInNetwork(ip, cidrBlock):
```

Comment:
@varxint : Can you verify this PR?

Reply:
this should be fine, we're just ignoring ACLs for traffic originating from microservices. We really should not have been imposing ACLs on that traffic from the beginning. The ACL excluded was the workaround to that but this is a more solid solution, as we discussed a while back on a call.

If @varxint has a minute to weigh in that would be great, otherwise we should merge and just double check before going to testing.

Reply:
Okay, so this was always the intent and just a change in methodology. No issues on my end then.

Reply:
yep exactly, thanks Chuck 👍